Recruit-Training/Embedded-Recruit-Training/Core/Src/GreenLedToggleTask.c

```diff
@@ -2,7 +2,13 @@
 
 void greenLedToggleTask(void const* arg)
 {
+    extern CAN_HandleTypeDef hcan2;
+    extern CAN_TxHeaderTypeDef CAN_TX;
+    extern uint8_t GreenToggleFlag;
+    
     //One time osDelayUntil initialization
+    static const uint32_t GREEN_LED_STATUS_STDID = 0xDDD;
+    static const uint32_t GREEN_LED_TOGGLE_FREQ = 10000;
```

Comment:
This number is wrong. It should be 10Hz. Use the formula `f=1/t`. Note that 1Hz is 1 second per cycle and we want it in milliseconds.